platforms/spring-boot/components-starter/camel-rest-openapi-starter/src/main/resources/META-INF/NOTICE.txt

```diff
@@ -0,0 +1,11 @@
+   =========================================================================
```

Comment:
Not sure we need this

Reply:
Both LICENSE.txt and NOTICE.txt are automatically generated and added into the git.

I checked other components starts, they also have those files in git, so I'd keep this new component same as others.